### DISCUSSION TITLE
mvn clean install fails

### DISCUSSION THREAD

**Discussion by mereolog** [finos/common-domain-model#3419] [Q&A] [answered]
## Support Question

`mvn clean install` fails with this error:

[ERROR] Failed to execute goal on project cdm-parent: Could not resolve dependencies for project org.finos.cdm:cdm-parent:pom:0.0.0.master-SNAPSHOT: Failed to collect dependencies at com.regnosys.rosetta:com.regnosys.rosetta:jar:9.8.0 -> org.eclipse.emf:org.eclipse.emf.ecore.xcore.lib:jar:1.6.0 -> org.eclipse.xtext:org.eclipse.xtext.xbase.lib:jar:2.27.0.M1: Failed to read artifact descriptor for org.eclipse.xtext:org.eclipse.xtext.xbase.lib:jar:2.27.0.M1: Could not transfer artifact org.eclipse.xtext:xtext-dev-bom:pom:2.27.0.M1 from/to regnosys-artifactory-snapshots (https://regnosys.jfrog.io/regnosys/libs-snapshot): authentication failed for https://regnosys.jfrog.io/regnosys/libs-snapshot/org/eclipse/xtext/xtext-dev-bom/2.27.0.M1/xtext-dev-bom-2.27.0.M1.pom, status: 401 Unauthorized

Here is my setting:

Apache Maven 3.8.2 (ea98e05a04480131370aa0c110b8c54cf726c06f)
Maven home: /Applications/apache-maven-3.8.2
Java version: 16.0.1, vendor: Oracle Corporation, runtime: .../jdk-16.0.1.jdk/Contents/Home
Default locale: en_GB, platform encoding: UTF-8
OS name: "mac os x", version: "11.3", arch: "x86_64", family: "mac"

**Comment by manel-martos** [comment]
Hi @mereolog,

I couldn't replicate the dependency resolution issue with the latest version on master branch on windows. 
Can you confirm whether you are interested in a specific CDM version/tag? Also confirm whether the issue has been solved on your local build.

You may need to skip test expectation issues with `mvn clean install -DskipTests` if you build the "tests" module.

Thanks,

```
Apache Maven 3.9.5 (57804ffe001d7215b5e7bcb531cf83df38f93546)
Java version: 17.0.9
Default locale: es_ES, platform encoding: Cp1252
OS name: "windows 11", version: "10.0", arch: "amd64", family: "windows"
```

**Comment by dschwartznyc** [comment, marked as answer]
I had a similar [issue building the Rune and CDM repos](https://github.com/finos/rune-dsl/issues/753) that was solved by following the recommendation to try:

`mvn clean install -U`

**Comment by mereolog** [comment]
This worked - thanks
